services/cron/setting.go

```diff
@@ -70,19 +68,19 @@ func (b *BaseConfig) DoNoticeOnSuccess() bool {
 }
 
 // FormatMessage returns a message for the task
-func (b *BaseConfig) FormatMessage(name, status string, doer *user_model.User, args ...interface{}) string {
+func (b *BaseConfig) FormatMessage(locale, name, status, doer string, args ...interface{}) string {
 	realArgs := make([]interface{}, 0, len(args)+2)
-	realArgs = append(realArgs, i18n.Tr("en-US", "admin.dashboard."+name))
-	if doer == nil {
+	realArgs = append(realArgs, i18n.Tr(locale, "admin.dashboard."+name))
+	if doer == "" {
 		realArgs = append(realArgs, "(Cron)")
 	} else {
-		realArgs = append(realArgs, doer.Name)
+		realArgs = append(realArgs, doer)
 	}
 	if len(args) > 0 {
 		realArgs = append(realArgs, args...)
 	}
-	if doer == nil || (doer.ID == -1 && doer.Name == "(Cron)") {
-		return i18n.Tr("en-US", "admin.dashboard.cron."+status, realArgs...)
+	if doer == "" {
+		return i18n.Tr(locale, "admin.dashboard.cron."+status, realArgs...)
 	}
-	return i18n.Tr("en-US", "admin.dashboard.task."+status, realArgs...)
+	return i18n.Tr(locale, "admin.dashboard.task."+status, realArgs...)
```

Comment:
Before, there was a status `aborted`, now, a new status `cancelled`.

The i18n keys would mismatch for these cases.

And I think we need some consts for these statuses. And we need some comments to remember that these status names are used for i18n keys.

Reply:
If you check the locale files you'll see that I've changed to cancelled to match the locale files ...

Reply:
Hmm ... strange, GitHub diff doesn't show the change from `dashboard.cron.aborted` to `dashboard.cron.cancelled`.

Just viewed the full file `options/locale/locale_en-US.ini`, `dashboard.cron.cancelled` is indeed there ..... maybe the old code's `aborted` was incorrect before.

No problem from my side, although I think a comment would help to clarify the relationship between these status names and i18n keys.

Reply:
The old codes aborted was indeed incorrect

Reply:
@wxiaoguang I've updated the comment of the function to add more information.